config/boards/rpi4b.wip

```diff
@@ -0,0 +1,80 @@
+# Raspberry Pi 4b
+enable_extension "flash-kernel"
+BOARD_NAME="RPi4b"
```

Comment:
BOARD_NAME="Raspberry Pi 4"

BTW. Will this build target also works OOB on Pi3 / Zero 2 and CM4?

Reply:
Done. 
I don't have any of these boards. CM4 should work, Pi3 I dunno.
I theory if Ubuntu works on a given RPi it should be easy.